ddl/db_test.go

```diff
@@ -519,6 +519,61 @@ func (s *testDBSuite) TestCancelAddIndex1(c *C) {
 	s.mustExec(c, "alter table t drop index idx_c2")
 }
 
+// TestCancelRenameIndex tests cancel ddl job which type is rename index.
+func (s *testDBSuite) TestCancelRenameIndex(c *C) {
```

Comment:
I think no need to do rollback of rename index. And the test can alse run pass in master without this PR.